skimmer/make_skims_pfnano_with_cross_section.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+MEMORY=10GB
+TIME=12:00:00
+PARTITION=standard
+CORES=2
+CHUNK_SIZE=1000
+N_WORKERS=150
+#EXECUTOR=dask/lpccondor    # HTCondor at LPC
+EXECUTOR=dask/slurm     # local job
```

Comment:
It would be nice to make a nice lean example for pfnano AOD. So what about removing `# local job`, this is not a local job.